composer/algorithms/mixup/README.md

```diff
@@ -80,13 +124,17 @@ Doing so avoids putting additional work on the CPU (since augmentation occurs on
 > MixUp requires a small amount of additional GPU compute and memory to produce the mixed-up batch.
 > In our experiments, we have found these additional resource requirements to be negligible.
 
-> ❗ MixUp Produces a Full Distribution, Not a Target Index
+> ❗ When `interpolate_loss=False` MixUp Produces a Full Distribution, Not a Target Index
 >
 > Many classification tasks represent the target value using the index of the target value rather than the full one-hot encoding of the label value.
-> Since MixUp interpolates between two target values for each example, it must represent the final targets as a dense distribution.
+> With `interpolate_loss=False` MixUp interpolates between two target values for each example, it must represent the final targets as a dense distribution.
```

Comment:
Also distribution -> vector of probabilities in the sentence right after this, and possibly elsewhere in the method card. Or define what we mean by "dense distribution" somewhere. I can ready this as referring to, e.g., a Gaussian, a Uniform, or basically any other continuous distribution, which makes this text seem confusing.

```suggestion
> With `interpolate_loss=False`, MixUp interpolates between two target values for each example, so it must represent the final targets as a dense vector of probabilities.
```

Reply:
Cleaned up the language here according to your suggestions.